.golangci.yml

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+run:
+  concurrency: 4
+  timeout: 5m
+  issues-exit-code: 0 # TODO: once all enabled linters are satisfied  
+  tests: false
+  skip-dirs:
+    (^|/).*traffic_router_golang.*($|/)
+  skip-dirs-use-default: true
+
+  output:
+  # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
+    format: colored-line-number
+    print-issued-lines: true
+    print-linter-name: true
+
+linters:
+  disable-all: true
+  enable:
+    - govet
+    - errcheck
+    - unused
+    - structcheck
+    - varcheck
+    - deadcode
+    - typecheck
+    - golint
+    - gofmt
```

Comment:
I'm not sure this is the best list of linters. In particular I would strongly encourage the use of staticcheck and take away unused (it was subsumed into staticcheck a while ago)? Also I suspect staticcheck does a better job at the checks that varcheck, deadcode, typecheck are trying to do.

Reply:
Here is the list of what staticcheck checks for: https://staticcheck.io/docs/checks

Reply:
Sounds great to me! Thanks for the input. Let me make that change and test it locally